src/cfnlint/helpers.py

```diff
@@ -290,15 +293,36 @@ def onerror(os_error):
         raise os_error
 
     for root, _, filenames in os.walk(directory, onerror=onerror):
-        for filename in fnmatch.filter(filenames, '[A-Za-z]*.py'):
-            pluginname = filename.replace('.py', '')
-            try:
-                fh, filename, desc = imp.find_module(pluginname, [root])
-                mod = imp.load_module(pluginname, fh, filename, desc)
-                result.extend(create_rules(mod))
-            finally:
-                if fh:
-                    fh.close()
+        if sys.version_info < (3,):
```

Comment:
split the changed code into a function `import_filename` or something? Then you can define it per version, e.g. https://github.com/scoutapp/scout_apm_python/blob/master/src/scout_apm/compat.py#L34

then you can have a single `for filename` loop :)